lib/testing-helpcenter/main_page_links/top_toolbar_links_constructor.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# Constructor for all links on main page for Installation Guides, Integration Guides, Administration Guides, User Guides, Developer Guides, Contribution Guides
+
+module TestingHelpCentreOnlyoffice
+  class TopToolbarLinksConstructor
+    include PageObject
+
+    def initialize(instance, title)
+      super(instance.webdriver.driver)
+      @instance = instance
+      @title_xpath = if user_guides_for_mobile(title.to_s)
+                       "//div[contains(@class, 'MobileApps')]/h1[contains(text(), '#{title}')]"
+                     else
+                       "//div[@id='toggleMenuEditors']/ul/li/h2[contains(text(), '#{title}')]"
+                     end
+      wait_to_load
+    end
+
+    def wait_to_load
+      @instance.webdriver.wait_until { title_element_present? }
+    end
+
+    def title_element_present?
+      @instance.webdriver.element_present?(@title_xpath)
+    end
+
+    def user_guides_for_mobile(title)
```

Comment:
`user_guides_for_mobile?` since this methods return only `true or false`

Reply:
Changed in commit [Method renaming](https://github.com/ONLYOFFICE-QA/testing-helpcenter/pull/19/commits/8c411c532d21416351baa7cb9c969ee96267fdfc)